source/release-notes/backend.txt

```diff
@@ -13,6 +13,15 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240807:
+
+07 August 2024 Release
+~~~~~~~~~~~~~~~~~~~~~~
+
+- Fixed bug where some destructive changes in push-pull were not being handled properly
```

Comment:
These seem to be all related to Sync, so maybe we want to put them under a "Device Sync" header? It looks like we have been inconsistent about this in the past - sometimes we have listed the specific part of the product that the release notes pertain to (i.e. Edge Server, CLI), but other times we don't.

This is probably a good high-level thing to consider as we're pondering our release notes process.

Reply:
Yep, makes sense!